### PR TITLE
a17-sidebar

### PR DESCRIPTION
**Pull request by ghost**
Creation of sidebar

### CODE REVIEW

File: src/jm/html-css/style.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+.info li {
+  margin-bottom: 10px;
+}
+
+.info .title {
+  font-weight: bold;
+}
+
```
Comment (Iregui44) [inline src/jm/html-css/style.css:196]: Theres is no hover pseduclass or anything that match the behaviour we expect when you hover over the logos.

File: src/jm/html-css/index.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                <span class="title">Sandra Marx</span><br />
+                <span class="subtitle">Sandra@gmail.com</span>
+              </li>
+              <li>
+                <i class="fa-solid fa-folder-open"></i>
+                <span class="title">Integration</span>
+              </li>
+              <li>
+                <i class="fa-solid fa-clock"></i>
+                <span class="title">Hostiry</span>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:165]:
```suggestion
                <span class="title">History</span>
```

**Review by Iregui44 — CHANGES_REQUESTED** [review]

File: src/jm/html-css/style.css (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css:196]: Added pseudoclass on hover

File: src/jm/html-css/index.html [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                <span class="title">Sandra Marx</span><br />
+                <span class="subtitle">Sandra@gmail.com</span>
+              </li>
+              <li>
+                <i class="fa-solid fa-folder-open"></i>
+                <span class="title">Integration</span>
+              </li>
+              <li>
+                <i class="fa-solid fa-clock"></i>
+                <span class="title">Histiry</span>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:165]:
```suggestion
                <span class="title">History</span>
```

File: html-css/index.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html lang="en">
+  <head>
```
Comment (Iregui44) [inline html-css/index.html:3]: What is this file?

File: src/jm/html-css/style.css [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+  padding: 5px;
+  position: absolute;
+  top: 50%;
+  left: 100%;
+  margin-left: 10px;
+  opacity: 0;
+  transition: opacity 0.3s;
+}
+
+.menu-item:hover .tooltip {
```
Comment (Iregui44) [inline src/jm/html-css/style.css:69]: hover is not working, when I hover over the icon the menu does not appear.

File: src/jm/html-css/index.html (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: ok

File: src/jm/html-css/index.html (hunk 3 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: ok

File: html-css/index.html (hunk 4 above)
Comment (Juanmediina) [reply html-css/index.html]: deleted file

File: src/jm/html-css/style.css (hunk 5 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css:78]: correction made

**Review by Iregui44 — APPROVED** [review]

**Comment by Iregui44** [comment]
You have conflicting errors please fix them to be able to marge

**Pull request closed without merging**